ansible/library/conn_graph_facts.py

```diff
@@ -279,12 +283,23 @@ def main():
     if not hostnames:
```

Comment:
Here the `hostnames` variable is not defined yet?

Reply:
It is assigned 2 lines above from argument.